apps/adminer.sh

```diff
@@ -32,7 +32,7 @@ then
     a2disconf adminer.conf
     rm -f $ADMINER_CONF
     rm -rf $ADMINERDIR
-    check_command apt-get purge adminer -y
+    check_command apt --purge autoremove -y adminer
```

Comment:
No, we use apt-get for purging.

Please revert. Thanks!

Reply:
done